homeassistant/components/universal/media_player.py

```diff
@@ -255,7 +255,8 @@ def state(self):
     @property
     def volume_level(self):
         """Volume level of entity specified in attributes or active child."""
-        return self._override_or_child_attr(ATTR_MEDIA_VOLUME_LEVEL)
+        level = self._override_or_child_attr(ATTR_MEDIA_VOLUME_LEVEL)
+        return float(level) if level is not None else None
```

Comment:
Please catch `ValueError` in case parsing goes wrong.

Reply:
I had to catch for `TypeError` (this is thrown when `float(None)` is executed). I've also caught `ValueError` in case non-float-like-strings are processed.